arch/arm64/boot/dts/qcom/sdm660-xiaomi-platina.dts

```diff
@@ -192,17 +192,17 @@
 
 	gpio_keys {
 		status = "okay";
-		compatible = "gpio-keys";
-		input-name = "gpio-keys";
-		#address-cells = <1>;
-		#size-cells = <0>;
+		compatible = "gpio-keys-polled";
```

Comment:
on mainline (>=5.16) keys work on lavender without polling btw. This will need to be reverted when moving to mainline kernel